crypto/rsa/rsa_asn1.c

```diff
@@ -67,12 +67,13 @@
 #include "internal.h"
 
 
-static int parse_integer(CBS *cbs, BIGNUM **out) {
+static int parse_integer_consttime(CBS *cbs, BIGNUM **out) {
```

Comment:
I guess that I misled you here. This should probably be named parse_integer_with_flags and it should take an `int flags` argument because...